blog/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.urls import reverse
+
+
+class User(AbstractUser):
+    ...
+
+
+class Post(models.Model):
+    title = models.CharField(max_length=255)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="posts",
+    )
+
+    def get_absolute_url(self):
+        return reverse("blog:post-detail", args=(str(self.id)))
```

Comment:
The `get_absolute_url` method should pass `args=[str(self.id)]` as a list instead of a tuple. This is a minor issue but should be corrected to avoid potential errors.